### PR TITLE
test uriencode

### PR DESCRIPTION
**Pull request by Kit486759**
Fixed issue #7 
[https://github.com/BlackStone8960/Musician-First/issues/7](url)

### CODE REVIEW

**Review by BlackStone8960 — APPROVED** [review]
It looks perfect! Thank you Kit :)

File: src/components/MusicianPage/MusicPlayer.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { ControlPointSharp } from '@material-ui/icons';
```
Comment (BlackStone8960) [inline src/components/MusicianPage/MusicPlayer.js:1]: Sorry, can you just delete this line before merging? This icon is not used in this component, right?

Comment (Kit486759) [reply src/components/MusicianPage/MusicPlayer.js:1]: Sure @BlackStone8960

**Comment by Kit486759** [comment]
Hi, @BlackStone8960  I have pushed the code again.

**Comment by BlackStone8960** [comment]
Good, thank you :)

**Review by miyabitanimchi — APPROVED** [review]
I checked your code and also checked that now the error doesn't occur. Thank you!!

**Pull request merged**